impeller/aiks/aiks_unittests.cc

```diff
@@ -2851,5 +2851,28 @@ TEST_P(AiksTest, CanCanvasDrawPicture) {
   ASSERT_TRUE(OpenPlaygroundHere(canvas.EndRecordingAsPicture()));
 }
 
+TEST_P(AiksTest, DrawPictureWithText) {
+  Canvas subcanvas;
+  ASSERT_TRUE(RenderTextInCanvas(
+      GetContext(), subcanvas,
+      "the quick brown fox jumped over the lazy dog!.?", "Roboto-Regular.ttf"));
+  auto picture = subcanvas.EndRecordingAsPicture();
```

Comment:
Please extend this test to render two text frames at different canvas scales in subcanvas. I don't think this patch renders this scenario correctly:

```
RenderTextInCanvas(...);
canvas.Translate(0, 100);
canvas.Scale(3, 3);
RenderTextInCanvas(...);
```

Reply:
So this works today but ends up with blurry text, because the _first_ call with a scale is winning.